### PR TITLE
Breakfast-Header test

### PR DESCRIPTION
**Pull request by ke-de-ve**
https://trello.com/c/7LzHiMUw/17-breakfasts-header-test

### CODE REVIEW

**Review by maximashuev — DISMISSED** [review]

File: Tests/BreakfastsPage_test.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from Pages.BreakfastPage import BreakfastPage
+
+
+class TestHomePage:
```
Comment (VictoriaRogo) [inline Tests/BreakfastsPage_test.py]: Why the class name is TestHomePage?

Comment (ke-de-ve) [reply Tests/BreakfastsPage_test.py]: renamed

File: Config/TestData.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class TestData:
 
     HOME_PAGE_URL = 'http://www.gidmenu.com'
+    BREAKFASTS_PAGE_URL = 'https://gidmenu.com/restaurants/zavtraki/'
```
Comment (Stan575) [inline Config/TestData.py]: you can use HOME_PAGE_URL to construct new page url

**Review by Stan575 — DISMISSED** [review]

**Review by VictoriaRogo — DISMISSED** [review]

File: Tests/BreakfastsPage_test.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    HOME_BUTTON = '//header//a[@class="navbar-brand"]'
+
+#    @pytest.mark.parametrize('restaurant', TestData.RESTAURANT_COLLECTION)
+    def test_title(self):
+        self.breakfasts_page = BreakfastPage(self.driver)
+        title = self.breakfasts_page.get_title()
+
+        assert title == BreakfastPage.TITLE
+        sleep(5)
```
Comment (Stan575) [inline Tests/BreakfastsPage_test.py]: Please delete sleep and import for sleep

File: Tests/BreakfastsPage_test.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+from Config.TestData import TestData
+from Pages.BreakfastPage import BreakfastPage
+
+
+class TestBreakfastPage:
+
+    HOME_BUTTON = '//header//a[@class="navbar-brand"]'
+
+#    @pytest.mark.parametrize('restaurant', TestData.RESTAURANT_COLLECTION)
```
Comment (Stan575) [inline Tests/BreakfastsPage_test.py]: Commented out code in repository is bad practice

**Review by Stan575 — CHANGES_REQUESTED** [review]

File: Config/TestData.py [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
 
     HOME_PAGE_URL = 'http://www.gidmenu.com'
+    BREAKFASTS_PAGE_URL = HOME_PAGE_URL + '/restaurants/zavtraki/'
+
+    PATRIARSHIE_PRUDY = {
+        'url': 'https://gidmenu.com/restaurants/patriarshie-prudy/?show_dish_rest_groups=N',
+        'title': 'Патриаршие Пруды'
+    }
+
+    RESTAURANT_COLLECTION = [PATRIARSHIE_PRUDY]
```
Comment (maximashuev) [inline Config/TestData.py:11]: Please delete lines 6-11.they are not related to this test.

**Review by maximashuev — CHANGES_REQUESTED** [review]

File: Config/TestData.py (hunk 5 above)
Comment (ke-de-ve) [reply Config/TestData.py:11]: they will be related later when page is fully implemented

Comment (maximashuev) [reply Config/TestData.py:11]: they are not part of your task and your TC
they should not be included here

**Pull request merged**